### DISCUSSION TITLE
Xbox Game Pass Crashing on start

### DISCUSSION THREAD

**Discussion by ChrisZaina** [moraroy/NonSteamLaunchers-On-Steam-Deck#101] [Q&A]
Hi there, I cannot get xbox game pass to work
Its running on experimental proton and just crashed on startup.
also tried every other proton type

thanks!

**Comment by Siegfriedmk** [comment]
Same for me. I suppose we have to use the Cloud version

**Reply by moraroy** [reply]
@ChrisZaina @Siegfriedmk do not run proton on the streaming sites.. they are opened with chrome which is Linux. Leave the compatability blank.
